docs/docsite/rst/porting_guides/porting_guide_base_2.10.rst

```diff
@@ -35,7 +35,10 @@ Playbook
 Command Line
 ============
 
-No notable changes
+* The ``ansible-galaxy login`` command has been removed, as the underlying API it used for GitHub auth is being shut down. Publishing roles or
+  collections to Galaxy via ``ansible-galaxy`` now requires that a Galaxy API token be passed to the CLI via the ``ANSIBLE_GALAXY_TOKEN``
```

Comment:
I don't believe `ANSIBLE_GALAXY_TOKEN` is used for authentication to Galaxy itself. This config option is meant to be your GitHub PAT used by `ansible-galaxy login` to then retrieve your Galaxy token. Very poorly named unfortunately but it was added when `login` was originally.

So the workarounds that I know of are:

* `~/.ansible/galaxy_token`
* `ansible.cfg` under the Galaxy server list configuration as documented at https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#configuring-the-ansible-galaxy-client
* Via `--token` to `ansible-galaxy`

Reply:
Nice- thanks for the clarification. I'll clean those up.

Reply:
While we are at it we may as well deprecate or at least put a comment on that config entry saying it isn't used anymore.